src/components/layout/Sidebar.vue

```diff
@@ -1,157 +1,139 @@
-      <md-list-item class="md-double-line">
-        <md-avatar>
-          <img :src="picture" alt="avatar">
-        </md-avatar>
-        <div class="sidebar-user-meta">
-          <div class="sidebar-user-fullname">{{ fullname }}</div>
-          <div class="sidebar-user-email">{{ email }}</div>
+  <div class="mdl-layout__drawer" id="sidebar">
+    <div class="sidebar-header">
+      <img src="../../assets/logo.svg" class="sidebar-logo" alt="">
+      <span class="mdl-layout-title sidebar-title">SYMPA</span>
+    </div>
+    <nav class="mdl-navigation sidebar-nav">
+      <hr class="sidebar-divider" v-if="logged">
+      <expand-list-two v-if="logged">
+        <div>
+          <img :src="picture" class="mdl-list__item-avatar" alt="avatar">
+          <span>{{ fullname }}</span>
+          <span class="mdl-list__item-sub-title">{{ email }}</span>
         </div>
-
-        <md-list-expand>
-          <md-list>
-            <md-list-item @click.native="logout">
+        <div slot="expand">
+          <div class="mdl-list__item expand-list-item" @click="logout">
+            <span class="mdl-navigation__link mdl-list__item sidebar-item mdl-list__item-primary-content">
               Logout
-            </md-list-item>
-          </md-list>
-        </md-list-expand>
-      </md-list-item>
-    </md-list>
-    <md-divider />
-    <md-list>
-      <md-list-item>
-        <router-link exact :to="{name: 'home'}">
-          <md-icon>dashboard</md-icon>
-          <span>Dashboard</span>
-        </router-link>
-      </md-list-item>
-      <md-list-item>
-        <router-link :to="{name: 'discussions'}">
-          <md-icon>forum</md-icon>
-          <span>Discussions</span>
-        </router-link>
-      </md-list-item>
-    </md-list>
-    <md-divider />
-    <md-list>
-      <md-list-item>
-        <router-link :to="{name: 'admin'}">
-          <md-icon>work</md-icon>
-          <span>Administration</span>
-        </router-link>
-      </md-list-item>
-      <md-list-item>
-        <router-link :to="{name: 'listmaster'}">
-          <md-icon>list</md-icon>
-          <span>Listmaster Admin</span>
-        </router-link>
-      </md-list-item>
-      <md-list-item>
-        <router-link :to="{name: 'profile'}">
-          <md-icon>person</md-icon>
-          <span>Profile</span>
-        </router-link>
-      </md-list-item>
-      <md-list-item>
-        <router-link :to="{name: 'settings'}">
-          <md-icon>settings</md-icon>
-          <span>Settings</span>
-        </router-link>
-      </md-list-item>
-      <md-list-item>
-        <router-link :to="{name: 'about'}">
-          <md-icon>help</md-icon>
-          <span>About/Help</span>
-        </router-link>
-      </md-list-item>
-    </md-list>
-  </md-sidenav>
+            </span>
+          </div>
+        </div>
+      </expand-list-two>
+      <hr class="sidebar-divider">
+      <router-link exact :to="{name: 'home'}" class="mdl-navigation__link mdl-list__item sidebar-item">
```

Comment:
https://github.com/sympa-community/sympa-vue/pull/2/commits/64a3d1a5fe300c8bf3bc194a554ed4c6a6c22e38 solved the cursor point : default -> pointer.